services/outputhost/kafkaStream.go

```diff
@@ -33,17 +33,37 @@ import (
 	"github.com/uber/cherami-thrift/.generated/go/store"
 )
 
+var (
+	errKafkaNilControlFlow = errors.New(`nil or non-positive controlFlow passed to Write()`)
+	errKafkaClosed = errors.New(`closed`)
+)
+
 // kafkaStream implements the same interface and similar-enough semantics to the Cherami store interface (BStoreOpenReadStreamOutCall)
 
 type kafkaStream struct {
 	creditSemaphore common.UnboundedSemaphore
 	kafkaMsgsCh     <-chan *s.ConsumerMessage
+	kConverter      KafkaMessageConverter
 	logger          bark.Logger
 	seqNo           int64
 }
 
-var errKafkaNilControlFlow = errors.New(`nil or non-positive controlFlow passed to Write()`)
-var errKafkaClosed = errors.New(`closed`)
+// KafkakMessageConverterConfig is used to config customized converter
+type KafkaMessageConverterConfig struct {
+	Destination *cherami.DestinationDescription
+	ConsumerGroup *cherami.ConsumerGroupDescription
```

Comment:
since we are not filling these in currently, probably better to comment them out for now.